### DISCUSSION TITLE
What is coming next ?

### DISCUSSION THREAD

**Discussion by TheoCouss** [Dokploy/dokploy#43] [General]
I was wondering what the roadmap is for this project?

**Comment by Siumauricio** [comment]
Hi.

Currently we have been collected a really cool features:

- Multi node support
- Docker compose support 
- Automatic dns generations
- DNS Wildcards

This is the board https://github.com/orgs/Dokploy/projects/3/views/2?layout=board

Currently we are working on Multi node support!

If you have more ideas about what we should implement would be awesome and we can think about how to implement

**Comment by TheoCouss** [comment]
The board you linked leads to a 404 :/

**Reply by Siumauricio** [reply]
my fault, check again https://github.com/orgs/Dokploy/projects/3/views/2?layout=board

**Reply by TheoCouss** [reply]
Thanks 😀
